config/root-config.in

```diff
@@ -469,16 +470,26 @@ while test $# -gt 0; do
       ;;
     --has-*)
       ### Check for feature
-      f=`echo $1 | sed 's/--has-//'`
-      for c in $features ; do
-         if test "x$c" = "x$f" ; then
+      target=`echo $1 | sed 's/--has-//'`
```

Comment:
Can you try what happens with `root-config --has-`
Is there a syntax error, do we say "yes", or do we handle that in a sane way?